gtda/sampling/sample_point_clouds.py

```diff
@@ -0,0 +1,177 @@
+"""Point-cloud–related plotting functions and classes."""
+# License: GNU AGPLv3
+
+import numpy as np
+from scipy.optimize import root_scalar
+
+from gtda.utils import validate_params
+from gtda.utils.intervals import Interval
+
+
+def sphere_sampling(n=1000, r=1, noise=0):
```

Comment:
you should use descriptive name. For example, r is for radius. Therefore, use radius as variable name.